frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -271,12 +271,24 @@ const HostSummary = ({
       platform,
       diskEncryptionEnabled
     );
+
     let statusText;
-    if (platform === "chrome") {
-      statusText = "Always on";
-    } else {
-      statusText = diskEncryptionEnabled ? "On" : "Off";
+    switch (true) {
```

Comment:
Because diskEncryptionEnabled is typed `any`, the component displaying "On" when it received the truthy value "---" from upstream.